packages/field-base/test/input-aria-mixin.test.js

```diff
@@ -0,0 +1,44 @@
+import { expect } from '@esm-bundle/chai';
+import sinon from 'sinon';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { InputAriaMixin } from '../src/input-aria-mixin.js';
+
+customElements.define(
+  'input-aria-mixin-element',
+  class extends InputAriaMixin(PolymerElement) {
+    static get template() {
+      return html`<slot name="label"></slot><slot name="input"></slot>`;
+    }
+  }
+);
+
+describe('input-aria-mixin', () => {
+  let element, input, label;
+
+  beforeEach(() => {
+    element = fixtureSync(`<input-aria-mixin-element></input-aria-mixin-element>`);
+    input = element.querySelector('[slot=input]');
+    label = element.querySelector('[slot=label]');
+  });
+
+  it('should set id attribute on the input', () => {
+    const idRegex = /^input-aria-mixin-element-\d$/;
+    expect(idRegex.test(input.getAttribute('id'))).to.be.true;
```

Comment:
```suggestion
    expect(input.getAttribute('id')).to.match(idRegex);
```

Reply:
Fixed and moved the test to `input-mixin.test.js`.